packages/@webex/webex-core/test/unit/spec/services/service-catalog.js

```diff
@@ -201,6 +201,56 @@ describe('webex-core', () => {
         assert.match(['example-a', 'example-b', 'example-c', 'example-e', 'example-f'], list);
       });
     });
+
+    describe('findServiceUrlFromUrl()', () => {
+      const otherService = {
+        defaultUrl: 'https://example.com/differentresource',
+        hosts: [{host: 'example1.com'}, {host: 'example2.com'}],
+      };
+
+      it.each([
+        'discovery',
+        'preauth',
+        'signin',
+        'postauth',
+        'override'
+      ])('matches a default url correctly', (serviceGroup) => {
```

Comment:
`serviceGroup`  doesn't seem to be used at all, so what's the point of these test cases? same for the next test

Reply:
got spot. The intention was to check that it would use services from each service group, not just preauth. I've fixed this now.